docs/datastorage/config.md

```diff
@@ -11,20 +11,29 @@ public class MyConfig {
   public static final MyConfig INSTANCE;
   public static final ForgeConfigSpec SPEC;
 
-  // There are other value types
+  // This is a config term
+  // There are other value types such as IntValue, EnumValue
   public final ForgeConfigSpec.BooleanValue setting;
     
   public MyConfig(ForgeConfigSpec.Builder builder) {
     setting = builder
       .comment("This is comment")
       .define("setting", false);
-    }
-    
+    //use defineEnum() to define enum type setting
+    //use defineInRange() to limit number range
+    //use translation() to specify translation key
+    //use worldRestart() to specify that world should reload when config changes
+    //You can also register nested config. Check the code of ForgeConfigSpec.Builder for more information
+  }
+  
+  // Read the config value
+  // Don't read it too early
```

Comment:
Client & Common configs are loaded after registry events and before common setup. Server configs are loaded when the world loads. You might want to mention this.